inst/extdata/get_attribute_values.xml

```diff
@@ -1,2 +1,2 @@
ample:CONUS_states>Montana</sample:CONUS_states></sample:CONUS_states><sample:CONUS_states gml:id="CONUS_States.26"><sample:CONUS_states>Nebraska</sample:CONUS_states></sample:CONUS_states><sample:CONUS_states gml:id="CONUS_States.27"><sample:CONUS_states>Nevada</sample:CONUS_states></sample:CONUS_states><sample:CONUS_states gml:id="CONUS_States.28"><sample:CONUS_states>New Hampshire</sample:CONUS_states></sample:CONUS_states><sample:CONUS_states gml:id="CONUS_States.29"><sample:CONUS_states>New Jersey</sample:CONUS_states></sample:CONUS_states><sample:CONUS_states gml:id="CONUS_States.30"><sample:CONUS_states>New Mexico</sample:CONUS_states></sample:CONUS_states><sample:CONUS_states gml:id="CONUS_States.31"><sample:CONUS_states>New York</sample:CONUS_states></sample:CONUS_states><sample:CONUS_states gml:id="CONUS_States.32"><sample:CONUS_states>North Carolina</sample:CONUS_states></sample:CONUS_states><sample:CONUS_states gml:id="CONUS_States.33"><sample:CONUS_states>North Dakota</sample:CONUS_states></sample:CONUS_states><sample:CONUS_states gml:id="CONUS_States.34"><sample:CONUS_states>Ohio</sample:CONUS_states></sample:CONUS_states><sample:CONUS_states gml:id="CONUS_States.35"><sample:CONUS_states>Oklahoma</sample:CONUS_states></sample:CONUS_states><sample:CONUS_states gml:id="CONUS_States.36"><sample:CONUS_states>Oregon</sample:CONUS_states></sample:CONUS_states><sample:CONUS_states gml:id="CONUS_States.37"><sample:CONUS_states>Pennsylvania</sample:CONUS_states></sample:CONUS_states><sample:CONUS_states gml:id="CONUS_States.38"><sample:CONUS_states>Rhode Island</sample:CONUS_states></sample:CONUS_states><sample:CONUS_states gml:id="CONUS_States.39"><sample:CONUS_states>South Carolina</sample:CONUS_states></sample:CONUS_states><sample:CONUS_states gml:id="CONUS_States.40"><sample:CONUS_states>South Dakota</sample:CONUS_states></sample:CONUS_states><sample:CONUS_states gml:id="CONUS_States.41"><sample:CONUS_states>Tennessee</sample:CONUS_states></sample:CONUS_states><sample:CONUS_states gml:id="CONUS_States.42"><sample:CONUS_states>Texas</sample:CONUS_states></sample:CONUS_states><sample:CONUS_states gml:id="CONUS_States.43"><sample:CONUS_states>Utah</sample:CONUS_states></sample:CONUS_states><sample:CONUS_states gml:id="CONUS_States.44"><sample:CONUS_states>Vermont</sample:CONUS_states></sample:CONUS_states><sample:CONUS_states gml:id="CONUS_States.45"><sample:CONUS_states>Virginia</sample:CONUS_states></sample:CONUS_states><sample:CONUS_states gml:id="CONUS_States.46"><sample:CONUS_states>Washington</sample:CONUS_states></sample:CONUS_states><sample:CONUS_states gml:id="CONUS_States.47"><sample:CONUS_states>West Virginia</sample:CONUS_states></sample:CONUS_states><sample:CONUS_states gml:id="CONUS_States.48"><sample:CONUS_states>Wisconsin</sample:CONUS_states></sample:CONUS_states><sample:CONUS_states gml:id="CONUS_States.49"><sample:CONUS_states>Wyoming</sample:CONUS_states></sample:CONUS_states></gml:featureMembers></wfs:FeatureCollection>
```

Comment:
schema location is still `gov.usgs.cida.gdp.derivative`. Should that be the case @dblodgett-usgs ?